onebusaway-android/build.gradle

```diff
@@ -91,6 +103,7 @@ android {
             manifestPlaceholders = [databaseAuthority: "com.joulespersecond.oba"]
             buildConfigField "int", "ARRIVAL_INFO_STYLE", "0" // Defined in BuildFlavorConstants
             buildConfigField "boolean", "USE_FIXED_REGION", "false" // Supports multi-region
+            buildConfigField "String", "APP_KEY", APP_KEY
```

Comment:
Assuming this is the embedded social API key, let's rename this accordingly to `EMBEDDEDSOCIAL_API_KEY`.  We have a number of API keys floating around.

Also, any reason why this field shouldn't exist in the other build flavors?  I'd like to allow agencies to white-label OBA entirely, including the embedded social features.  This would obviously require their own API key, but the feature should still exist.